build/commands/lib/util.js

```diff
@@ -545,7 +545,7 @@ const util = {
   // So, this copying in every build doesn't affect compile performance.
   updateMidlFiles: () => {
     Log.progressScope('update midl files', () => {
-      for (const source of ["google_update", "brave"]) {
+      for (const source of ["google_update", "brave", "chrome"]) {
```

Comment:
shouldn't we just copy all of `win_build_output`?

Reply:
I don't think we want to do that; there are some other ones there by Chromium which we're guaranteed not to have touched
![image](https://github.com/brave/brave-core/assets/4733304/12ccf122-24c8-40f0-90a7-3d5e43a55182)

Reply:
but those wouldn't be in `brave/win_build_output/midl`, right? Do we have things in that directory that shouldn't be copied?

Reply:
ah - yes, you're right! We can change this to auto-copy everything found 😄

Reply:
Fixed in https://github.com/brave/brave-core/pull/20754/commits/c9f9ec6b22aef1f81e971ecc17055c9afb10ca08